src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/UserControllerIntTest.kt

```diff
@@ -263,7 +263,7 @@ class UserControllerIntTest : IntegrationTestBase() {
     @Test
     fun `My email endpoint returns unverified email`() {
       val username = "USER_UNVERIFIED"
-      hmppsAuthMockServer.stubUserEmail(username, true, false)
+      hmppsAuthMockServer.stubUserEmail(username, unverifiedParam = true, verifiedEmail = false)
```

Comment:
Added from previous PR - to improve clarity (and any avoid errors as 2 boolean params)